### Урок 2. Дискретные случайные величины. Закон распределения вероятностей. Биномиальный закон распределения. Распределение Пуассона

#### Задание №1
Случайная непрерывная величина A имеет равномерное распределение на промежутке (200, 800].
Найдите ее среднее значение и дисперсию.

In [1]:
# Средее велечины А
mean_a = (800-200)/2
mean_a

300.0

In [2]:
# Дисперсия
var_a = (800-200)**2 / 12
var_a

30000.0

#### Задание №2
О случайной непрерывной равномерно распределенной величине B известно, что ее дисперсия равна 0.2.
Можно ли найти правую границу величины B и ее среднее значение зная, что левая граница равна 0.5?
Если да, найдите ее.

D(B) = (b-a)^2/12

12*D(B) = b^2-2*a*b+a^2

b^2 - 2*a*b + (a^2-12*D(B)) = 0

b = (2*a +(-) ((-2*a)^2 - 4 * (a^2-12*D(B)))^0.5) / 2

In [9]:
# Правая граница величены B
b1 = (2*0.5 + ((-2*0.5)**2 - 4*(0.5**2 - 12*0.2))**0.5 ) / 2
b1

2.049193338482967

In [11]:
b2 = (2*0.5 - ((-2*0.5)**2 - 4*(0.5**2 - 12*0.2))**0.5 ) / 2
b2

-1.0491933384829668

b2 < a - значит корня не существует

Правая граница равна 2.049193338482967

#### Задание №3 
Непрерывная случайная величина X распределена нормально и задана плотностью распределения
f(x) = (1 / (4 * sqrt(2*pi))) * (exp(-((x+2)**2) / 32))
Найдите:
а). M(X)
б). D(X)
в). std(X) (среднее квадратичное отклонение)

а). M(X) = -2

б). D(X) = 16

в). std(X) = 4

#### Задание №4
Рост взрослого населения города X имеет нормальное распределение.
Причем, средний рост равен 174 см, а среднее квадратичное отклонение равно 8 см.
Какова вероятность того, что случайным образом выбранный взрослый человек имеет рост:
а). больше 182 см
б). больше 190 см
в). от 166 см до 190 см
г). от 166 см до 182 см
д). от 158 см до 190 см
е). не выше 150 см или не ниже 190 см
ё). не выше 150 см или не ниже 198 см
ж). ниже 166 см.

In [12]:
# а). больше 182 см
Z = (174- 182) / 8
Z

-1.0

Z = - 1

Вероятность того, что случайным образом выбранный взрослый человек имеет рост больше 182 см P < 15.87%

In [13]:
# б). больше 190 см
Z = (174- 190) / 8
Z

-2.0

Z = - 2

P < 2.28%

In [15]:
# в). от 166 см до 190 см
Z1 = (166 - 174) / 8
Z1 

-1.0

In [16]:
Z2 = (190 - 174) / 8
Z2 

2.0

Z1 = -1    P1 = 15.87%

Z2 = 2     P2 = 97.72%

P общее = P2 - P1 = 81.85%

In [17]:
# г). от 166 см до 182 см
Z2 = (182 - 174) / 8
Z2 

1.0

Z1 = -1    P1 = 15.87%

Z2 = 1     P2 = 84.13%

P общее = P2 - P1 = 68.26%

In [20]:
# д). от 158 см до 190 см 
Z1 = (158 - 174) / 8
Z1 

-2.0

Z1 = -2    P1 = 2.28%

Z2 = 2     P2 = 97.72%

P общее = P2 - P1 = 95.44%

In [21]:
# е). не выше 150 см или не ниже 190 см
Z1 = (150 - 174) / 8
Z1 

-3.0

In [ ]:
Z1 = -3 P1 = 0.14%

Z2 = -2 P2 = 2.28%

P общее = P2 + P1 = 2.42%


In [22]:
# ё). не выше 150 см или не ниже 198 см
Z2 = (174 - 198) / 8
Z2 

-3.0

P общее = P2 + P1 = 0.28%

In [24]:
# ж). ниже 166 см.
Z = (166 - 174) / 8
Z

-1.0

P < 15.87%

#### Задание №5
На сколько сигм (средних квадратичных отклонений) отклоняется рост человека, равный 190 см, от
математического ожидания роста в популяции, в которой M(X) = 178 см и D(X) = 25 кв.см?

In [26]:
std_x = 25**0.5
std_x

5.0

In [29]:
std_man = (190 - 178)/std_x
std_man

2.4